starboard/android/shared/android_main.cc

```diff
@@ -41,6 +41,9 @@
 namespace starboard {
 namespace android {
 namespace shared {
+
+atomic_bool g_block_swapbuffers;
```

Comment:
Does this need to be an atomic ? It's only written to from main thread and read in renderer thread, it can just be a `bool` ?

Reply:
It has to be atomic exactly because it's read from a different thread than from where it's written. It ensures that it's not read while being written and potentially only partially written, in addition to using memory barriers to make sure it's not using a stale (cached) value.

Reply:
But there are no memory barriers here. Just making it atomic doesn't guarantee it's correctly reflected between threads. If you want to absolutely ensure it's correct, just use a lock.

Reply:
The `load()` and `store()` of atomic_bool should include barriers as needed.